CHANGELOG.md

```diff
@@ -15,7 +14,8 @@ Performance:
 Internal changes:
 
 * The launcher is now part of the TruffleRuby repository, rather than part of
-the GraalVM repository.
+the GraalVM repository and it accepts `--native` and similar options in 
+`TRUFFLERUBYOPT` environment variable.
```

Comment:
The `--native` in TRUFFLERUBYOPT is not an internal change but a new feature or bug fix.
"in *the* TRUFFLERUBYOPT` environment variable"